packages/app/src/cli/models/app/loader.test.ts

```diff
@@ -2828,7 +2806,7 @@ describe('WebhooksSchema', () => {
       code: zod.ZodIssueCode.custom,
       message: 'You can’t have duplicate subscriptions with the exact same `topic`, `uri` and `filter`',
       fatal: true,
-      path: ['webhooks', 'subscriptions', 1, 'topics', 0, 'products/create'],
+      path: ['webhooks', 'subscriptions', 0, 'topics', 1, 'products/create'],
```

Comment:
ohh, I think this might have something to do with condensing the webhooks, instead of sorting? cause the subscriptions look like this after the transform on the schema:

```
      subscriptions: [
        {
          topics: ['products/update', 'products/update'],
          uri: 'https://example.com',
          filter: 'title:shoes',
        },
      ],
```